lib/views/widgets/shared/search_input_widget.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/material.dart';
```

Comment:
Is it generic search inout widget or used in any of screen ?
Changing file name to more meaningful, can help to understand what it is for.

Reply:
Not now but there is a page (for Hashtag searching) to be added in GSOC probably where we can use this widget.

Reply:
this file contains a text-input widget decorated for searching, what should I name it?

Reply:
@rutvik11062000 if you are not using this widget then we can remove this code and add when required later.

Reply:
I'm using it in join-organization-page.

Reply:
Sir please check latest commit I've renamed it to "SearchTextInputWidget".

Reply:
Looking good, there was a spelling error and one area I was thinking you could also update to have the retrieveException function. Once updated I will approve.